Vagrantfile

```diff
@@ -10,7 +10,8 @@ Vagrant.configure("2") do |config|
   config.vm.synced_folder "~/.hab/cache/keys", "/hab/cache/keys"
   config.vm.synced_folder "~/.hab/etc", "/hab/etc"
 
-  config.vm.network "forwarded_port", guest: 80, host: 9636
+  config.vm.network "forwarded_port", guest: 80, host: 80
```

Comment:
That would require users to run Vagrant/VirtualBox as root to work. I think it should be mapped to an unprivileged port instead.

But I'm not sure if that forwarding is even needed; what is listening on port 80 in the guest?

Reply:
@schu Agree. I would just remove the line with port 80.